WebApp/client/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "webclient",
+  "version": "3.0.0",
+  "private": true,
+  "scripts": {
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --colors test/*.js"
```

Comment:
**cross-env** package has not been maintained in this repository.
We should avoid to use the package.
https://github.com/kentcdodds/cross-env